shiny/session/_session.py

```diff
@@ -334,6 +335,7 @@ def verify_state(expected_state: ConnectionState) -> None:
                 ...
             except Exception as e:
                 try:
+                    traceback.print_exception(*sys.exc_info())
```

Comment:
Without this fix, errors in the InputHandler stack kill the session without giving a traceback.

Reply:
If this `sys.exc_info` is needed only for Python 3.9 and below, can you add a comment saying that, and saying what it can be changed to in the future, when we drop 3.9 support?
https://docs.python.org/3/library/traceback.html#traceback.print_exception